src/pynxtools/testing/nexus_conversion.py

```diff
@@ -134,10 +140,9 @@ def check_reproducibility_of_nexus(self):
             "DEBUG - value: v",
             "DEBUG - value: https://github.com/FAIRmat-NFDI/nexus_definitions/blob/",
         ]
-        ref_log = get_log_file(self.ref_nexus_file, "ref_nexus.log", self.tmp_path)
         gen_log = get_log_file(self.created_nexus, "gen_nexus.log", self.tmp_path)
```

Comment:
```suggestion
        gen_log_file = get_log_file(self.created_nexus, "gen_nexus.log", self.tmp_path)
```
To having the same name convention.

Reply:
ok